src/effects.js

```diff
@@ -27,19 +27,27 @@ var
 	rfxtypes = /^(?:toggle|show|hide)$/,
 	rrun = /queueHooks$/;
 
-function raf() {
+function raf( timestamp ) {
 	if ( timerId ) {
 		window.requestAnimationFrame( raf );
-		jQuery.fx.tick();
+		jQuery.fx.tick( timestamp );
 	}
 }
 
+// We need to be using jQuery.now() or performance.now() consistently as they return different
+// values: performance.now() counter starts on page load.
+function getTimestamp() {
+    return window.performance && typeof window.performance.now === "function" ?
```

Comment:
> In what browsers we do not have now in performance object?

IE 9, Android <4.4, iOS 7.

And Node.js. :-)

> Also, could you try to use if..else instead of a ternary?

Sure.